RadixWallet/Clients/DeviceFactorSourceClient/DeviceFactorSourceClient+Live.swift

```diff
@@ -79,6 +79,22 @@ extension DeviceFactorSourceClient: DependencyKey {
 			return entities.contains(where: { !$0.isMnemonicPresentInKeychain })
 		}
 
+		let unrecoverableEntitiesCount: @Sendable () async throws -> (accounts: Int, personas: Int) = {
```

Comment:
This may go away if we don't need to be smart about amount of entities that are unrecoverable. Slack [thread](https://rdxworks.slack.com/archives/C03QFAWBRNX/p1715679454159999?thread_ts=1715353183.606979&cid=C03QFAWBRNX)